### DISCUSSION TITLE
Trpc fetch adapter cors solutions?

### DISCUSSION THREAD

**Discussion by Eduard-Hasa** [trpc/trpc#3112] [Q&A]
Im working with cloudflare workers and wondering if anyone knew the best approach to solving a cors issue? I have seen the documentation and another users question but that was all regarding a next request handler and kind of confused how to apply that to the Trpc fetchRequestHandler as they seem very different.

Any advice is greatly appreciated.

**Comment by andrewtpoe** [comment]
Struggling with the same issue here. Did you find any resolution?

**Comment by Dxig** [comment]
What about using nginx to support cors?

**Reply by andrewtpoe** [reply]
This won't work for my project. I'm building a serverless app based cloudflare's workers. As best as I can tell, there is no way to use nginx from within workers. To include nginx I'd have to set up an instance elsewhere and proxy requests through to cloudflare.

I'm not super familiar with cloudflare workers yet, my project is to get experience. Feel free to offer guidance if you know something I don't.
